Countries Info/ViewModels/CountriesListViewModel.swift

```diff
@@ -6,31 +6,41 @@
 //
 
 import Foundation
+import Combine
 
 protocol CountriesListViewModelProtocol {
     var countries: [Country]? { get }
```

Comment:
remove this from protocol, as out of ViewModel we should know just for Publisher.